server/controllers/api/apiController.ts

```diff
@@ -83,11 +83,11 @@ export default class ApiController {
       } else {
         configuredAddress = `${address.buildingName ? convertToTitleCase(address.buildingName) : ''} ${
           address.addressNumber ? address.addressNumber : ''
-        } ${address.streetName ? convertToTitleCase(address.streetName) : ''}
```

Comment:
This tidies up the display of the address

Reply:
What's change here out of the linting?